saleor/graphql/core/types.py

```diff
@@ -33,20 +33,35 @@ class Error(graphene.ObjectType):
     message = graphene.String()
 
 
-class Price(graphene.ObjectType):
+class Money(graphene.ObjectType):
+    currency = graphene.String()
+    amount = graphene.Float()
+    localized = graphene.String()
+
+    def resolve_localized(self, info):
+        return prices_i18n.amount(self)
+
+
+class TaxedMoney(graphene.ObjectType):
     currency = graphene.String()
     gross = graphene.Float()
```

Comment:
Why aren't `gross` and `net` fields of type `Money` you define above? This would also take core of currency and currency exchange.